<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtensionDialogAdd</class>
 <widget class="QDialog" name="ExtensionDialogAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un groupe d'extensions</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom du groupe d'extensions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Liste des extensions</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewExtensionGroup">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dialogOkBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dialogAnnulerBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogAnnulerBtn</sender>
   <signal>clicked()</signal>
   <receiver>ExtensionDialogAdd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
